src/atoms/Br/__tests__/br.spec.js

```diff
@@ -1,17 +1,9 @@
 import React from 'react';
-import { mount } from 'enzyme';
 import renderer from 'react-test-renderer';
 
 import Br from 'atoms/Br';
 
 describe('<Br />', () => {
-
```

Comment:
`jest/no-identical-title` was throwing errors because each of these files had two identical test names. discussed with john prior to making any changes - decided to get rid of the mounted component tests in each file.